eng/Version.Details.xml

```diff
@@ -1,10 +1,10 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="7.0.201-rtm.23116.12" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+     <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="7.0.203-servicing.23170.23" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
```

Comment:
this just makes sure everything is building with the correct sdk

Reply:
If you fixed this manually, do you just want to use the latest public release `7.0.203`?

https://dotnet.microsoft.com/download/dotnet/7.0